cwms/__init__.py

```diff
@@ -7,6 +7,9 @@
 from .levels.location_levels import *
 from .levels.specified_levels import *
 from .locations.physical_locations import *
+from .projects.project_lock_rights import *
```

Comment:
since you made this pull request the _init file structure changed.  please update to cwms.projects....  This is for compatibility with airflow. after that I will approve.

Reply:
I rebased and updated the _init

Reply:
can you add project_locks and project lock rights as well.  looks like you just have projects on the init now.

Reply:
added